hardhat.config.ts

```diff
@@ -32,17 +33,16 @@ const settings = {
 };
 
 const config: HardhatUserConfig = {
-  // networks: {
-  //   hardhat: {
-  //     forking: {
-  //       url: `https://mainnet.infura.io/v3/${INFURA_API_KEY}`
-  //     },
-  //   },
-  //   ropsten: {
-  //     url: `https://ropsten.infura.io/v3/${INFURA_API_KEY}`,
-  //     accounts: [`0x${WALLET_PRIVATE_KEY}`],
-  //   },
```

Comment:
@byterose do we still need this or its working fully through wallet connection? which should, in the same time for having this as an extra option we can just keep it commented @ElmoArmy